Dockerfile

```diff
@@ -212,7 +213,8 @@ COPY "startup/${LABKEY_DISTRIBUTION}.properties" \
     startup/49_distribution.properties
 
 # add logging config files
-COPY log4j2.xml log4j2.xml
+#COPY log4j2.xml log4j2.xml
```

Comment:
@labkey-willm do we need this? since we want to use the `jar` version?

Reply:
we do not! yay!